src/main/java/ch/njol/skript/effects/EffBroadcast.java

```diff
@@ -31,29 +36,32 @@
 import ch.njol.skript.lang.VariableString;
 import ch.njol.skript.registrations.Classes;
 import ch.njol.skript.util.LiteralUtils;
+import ch.njol.skript.util.SkriptColor;
+import ch.njol.skript.util.Utils;
 import ch.njol.skript.util.chat.BungeeConverter;
 import ch.njol.skript.util.chat.ChatMessages;
 import ch.njol.util.Kleenean;
+import ch.njol.util.StringUtils;
 import ch.njol.util.coll.CollectionUtils;
 import net.md_5.bungee.api.chat.BaseComponent;
 import org.bukkit.Bukkit;
 import org.bukkit.World;
 import org.bukkit.command.CommandSender;
 import org.bukkit.event.Event;
-import org.eclipse.jdt.annotation.Nullable;
-
-import java.util.ArrayList;
-import java.util.List;
+import org.bukkit.event.server.BroadcastMessageEvent;
+import org.jetbrains.annotations.Nullable;
 
 @Name("Broadcast")
 @Description("Broadcasts a message to the server.")
 @Examples({
 	"broadcast \"Welcome %player% to the server!\"",
 	"broadcast \"Woah! It's a message!\""
 })
-@Since("1.0, 2.6 (broadcasting objects), 2.6.1 (using advanced formatting)")
+@Since("1.0, 2.6 (broadcasting objects), 2.6.1 (using advanced formatting), INSERT VERSION (calls broadcast event)")
```

Comment:
I don't believe we need a version tag here, the pr this mostly affects isn't merged, the only time I'd believe this would matter is if we're taking external interacts into account such as plugins.

Reply:
Marking this as resolved due to another review of this approves this change.